version/version.go

```diff
@@ -19,6 +20,44 @@ const (
 	NoMinorFormat = "x"
 )
 
+// Enum that specifies which version component should be bumped
+type TargetRelease int
```

Comment:
I think I would prefer something like `VersionComp` and
```go
const (
    Patch VersionComp = iota
    Minor
    Major
)
```

Reply:
Again, not a native speaker, so I have no reason not to accept your proposal.